### PR TITLE
Adding a doc about how to install the meshctl CLI

### PR DESCRIPTION
**Pull request by artberger**
So that we can point to this page from the many other places in the doc that we have variants of these steps. Encountered this while trying to verify the upgrade steps.

### CODE REVIEW

File: docs/content/reference/cli/_index.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ description: |
   Detailed descriptions and options for working with the Gloo Mesh CLI. 
 weight: 2
 ---
-This section contains generated reference documentation for the `Gloo Mesh` CLI.
+This section contains generated reference documentation for the `Gloo Mesh` CLI.
```
Comment (saiskee) [inline docs/content/reference/cli/_index.md:8]:
```suggestion
This section contains generated reference documentation for the Gloo Mesh CLI, meshctl. To install the CLI, see [meshctl CLI]({{< versioned_link_path fromRoot="/setup/installation/meshctl_cli_install/" >}}).
```

File: docs/content/reference/cli/_index.md [hunk 2]
```diff
@@ -5,4 +5,4 @@ description: |
   Detailed descriptions and options for working with the Gloo Mesh CLI. 
 weight: 2
 ---
-This section contains generated reference documentation for the `Gloo Mesh` CLI.
\ No newline at end of file
+This section contains generated reference documentation for the Gloo Mesh CLI, meshctl. To install the CLI, see [meshctl CLI]({{< versioned_link_path fromRoot="/setup/installation/meshctl_cli_install/" >}}).
```
Comment (artberger) [inline docs/content/reference/cli/_index.md:8]: The codegen failed...maybe it needs an extra line? will try that.

From [logs](https://github.com/solo-io/gloo-mesh/pull/1861/checks?check_run_id=3473319866#step:8:3065)
```
--- a/docs/content/reference/cli/_index.md
+++ b/docs/content/reference/cli/_index.md
@@ -5,4 +5,4 @@ description: |
   Detailed descriptions and options for working with the Gloo Mesh CLI. 
 weight: 2
 ---
-This section contains generated reference documentation for the Gloo Mesh CLI, meshctl. To install the CLI, see [meshctl CLI]({{< versioned_link_path fromRoot="/setup/installation/meshctl_cli_install/" >}}).
\ No newline at end of file
+This section contains generated reference documentation for the Gloo Mesh CLI, meshctl. To install the CLI, see [meshctl CLI]({{< versioned_link_path fromRoot="/setup/installation/meshctl_cli_install/" >}}).
+ exit 1
Error: Process completed with exit code 1.
```

Comment (artberger) [reply docs/content/reference/cli/_index.md:8]:
```suggestion
This section contains generated reference documentation for the Gloo Mesh CLI, meshctl. To install the CLI, see [meshctl CLI]({{< versioned_link_path fromRoot="/setup/installation/meshctl_cli_install/" >}}).

```

**Comment by Rachael-Graham** [comment]
LGTM for docs' side

**Review by saiskee — APPROVED** [review]

**Pull request merged**